index.bs

```diff
@@ -3861,6 +3864,45 @@ This [=registration extension=] and [=authentication extension=] enables use of
                     01           -- Subitem 3: CBOR short for Matcher Protection Type Software
     </pre>
 
+
+## Biometric Authenticator Performance Bounds Extension (biometricBound) ## {#sctn-authenticator-biometric-criteria-extension}
+
+This [=registration extension=] allows [=[RPS]=] to specify the desired performance bounds for selecting [=biometric authenticators=]
+as candidates to be employed in a [=registration=] ceremony. 
+
+: Extension identifier
+:: `biometricMatchingCriteria`
+
+: Client extension input
+:: Biometric performance bounds:
+
+    <pre class="idl">
+    dictionary authenticatorBiometricPerfBounds{
+        float FAR;
+        float FRR;
+        };
+    </pre>
+
+    The FAR is the maximum false acceptance rate for a biometric authenticator allowed by the [=[RP]=].
+
+    The FAR is the maximum false rejection rate for a biometric authenticator allowed by the [=[RP]=].
+
+: Client extension processing
+:: This extension can only be used during {{CredentialsContainer/create()}}. If the client supports this
+    extension, it MUST NOT use a biometric authenticator whose FAR or FRR does not match the bounds as provided.
```

Comment:
How does client know these values? Are you expecting client to contact some authority who verifies it? FIDO metadata service?

Reply:
@akshayku 

Yes, that is certainly one potential approach.  Expectation is that authenticator would've gone through FIDO Biometric Cert, had its performance verified, and agreed to publish its performance data on the FIDO Metadata Service (MDS).

Reply:
This whole thing seems really brittle.  It's more likely to provide a false sense of security than to actually be accurate.

Reply:
so the above text perhaps ought to additionally indicate that various platform-specific means (eg some possibly based on the MDS) are anticipated for the client platform to use in order to have knowledge of these values.